tests/utils.py

```diff
@@ -1,7 +1,7 @@
 import contextlib
 import datetime
 import functools
-from unittest import mock
+from unittest import mock, SkipTest
```

Comment:
This might be some left-over from conflicts resolving, I will check and remove it.